HW_7/functional.py

```diff
@@ -0,0 +1,67 @@
+# Functional programming
+
+
+def sequential_map(*args):
+    *funcs, elements = args
+    mapped_elements = list()
+    for element in elements:
+        res = element
+        for func in funcs:
+            res = func(res)
+        mapped_elements.append(res)
+    return mapped_elements

```

Comment:
Сложно, у нас тут функциональное программирование, поэтому давай просто применим `map`
```suggestion
    for func in funcs:
        elements = map(func, elements)
    return list(elements)
```